dtfs-central-api/src/services/repositories/durable-functions-repo.test.ts

```diff
@@ -0,0 +1,72 @@
+import db from '../../drivers/db-client';
+import { DB_COLLECTIONS } from '../../constants';
+import { deleteAllAcbsDurableFunctionLogs, deleteAllDurableFunctionLogs } from './durable-functions-repo';
+
+describe('durable-functions-repo', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const deleteTestCases = [
+    {
+      testCase: 'deleteAllDurableFunctionLogs',
+      makeRequest: deleteAllDurableFunctionLogs,
+      expectedDeleteManyCalledWith: {},
+    },
+    {
+      testCase: 'deleteAllAcbsDurableFunctionLogs',
+      makeRequest: deleteAllAcbsDurableFunctionLogs,
+      expectedDeleteManyCalledWith: { type: { $eq: 'ACBS' } },
```

Comment:
* Need to add `status: Completed` in an `$and[]` compound Mongo query please.
* `ACBS` should be picked up from the constant please.